src/test/documentation_examples.rs

```diff
@@ -1414,6 +1422,75 @@ async fn versioned_api_examples() -> Result<()> {
     let client = Client::with_options(options)?;
     // End Versioned API Example 4
 
+    let mut options = ClientOptions::parse(&uri).await?;
+    let server_api = ServerApi::builder()
+        .version(ServerApiVersion::V1)
+        .strict(true)
+        .build();
+    options.server_api = Some(server_api);
+    let client = Client::with_options(options)?;
+    let db = client.database("versioned-api-migration-examples");
+    db.collection::<Document>("sales").drop(None).await?;
+
+    use std::{error::Error, result::Result};
+    // Start Versioned API Example 5
```

Comment:
It might be more useful to the docs team if we number these examples Count Migration 1-4 to stay in line with the ticket description.

Reply:
There was a [comment](https://jira.mongodb.org/browse/DRIVERS-1846?focusedCommentId=3950144&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3950144) in the drivers ticket that these should be "Versioned API Example 5" and onwards, did I miss a convention for them to be elsewise?

Reply:
Nope, I missed that comment!